src/coreclr/gc/gcpriv.h

```diff
@@ -124,7 +124,7 @@ inline void FATAL_GC_ERROR()
 
 #define CARD_BUNDLE         //enable card bundle feature.(requires WRITE_WATCH)
 
-// #define ALLOW_REFERENCES_IN_POH  //Allow POH objects to contain references.
+#define ALLOW_REFERENCES_IN_POH  //Allow POH objects to contain references.
```

Comment:
Is this going to have negative performance effect on POH?

Reply:
it will, a little bit. I do plan to improve this in the future so we don't take this perf hit. but the positive perf impact of this was quite significant so we are willing to take this small regression. 

---
In reply to: [567187855](https://github.com/dotnet/runtime/pull/47651#discussion_r567187855) [](ancestors = 567187855)

Reply:
Can we quantify the perf hit here? Since we expect to backport this to 5 servicing release, need to ensure it doesn't cause measurable regression for certain apps.